### DISCUSSION TITLE
installation error

### DISCUSSION THREAD

**Discussion by haizhou0410** [mdolab/idwarp#93] [General]
Hi, I met some problem when compile. Could someone give me some advice? Thank you!

make[3]: Entering directory '/home/haizhou/packages/idwarp-main/src/modules'
mpifort -I../..//mod -I/home/haizhou/packages/CGNS-4.4.0/opt-gfortran/include -fPIC -fdefault-real-8 -O2 -std=f2008 -I/home/haizhou/packages/petsc-3.20.1/include -I/home/haizhou/packages/petsc-3.20.1/real-debug/include -I/home/haizhou/packages/openmpi-5.0.0/opt-gfortran/include -I/home/haizhou/packages/petsc-3.20.1/ -c kd_tree.F90 -o ../..//obj/kd_tree.o
kd_tree.F90:1568:49:

 1568 |         call VecGetArrayF90(XsLocal, XsPtr, ierr)
      |                                                 1
Error: Type mismatch in argument ‘array’ at (1); passed REAL(8) to COMPLEX(8)
kd_tree.F90:1575:53:

 1575 |         call VecRestoreArrayF90(XsLocal, XsPtr, ierr)
      |                                                     1
Error: Type mismatch in argument ‘array’ at (1); passed REAL(8) to COMPLEX(8)
make[3]: *** [../..//Rules.mk:2: kd_tree.o] Error 1
make[3]: Leaving directory '/home/haizhou/packages/idwarp-main/src/modules'
make[2]: *** [Makefile:44: all] Error 2
make[2]: Leaving directory '/home/haizhou/packages/idwarp-main/src/modules'
make[1]: *** [Makefile:55: warp] Error 1
make[1]: Leaving directory '/home/haizhou/packages/idwarp-main'
make: *** [Makefile:24: default] Error 2

**Comment by Mohsin-Bashir** [comment]
Hi @haizhou0410 how did you solve this issue?
